subprojects/docs/src/docs/userguide/workingWithDependencies.adoc

```diff
@@ -46,13 +46,25 @@ You can iterate over the complete set of artifacts resolved for a module with th
 Iterating over the artifacts of a module automatically resolves the configuration. A resolved configuration becomes immutable and cannot add or remove dependencies. If needed you can copy a configuration for further modification via api:org.gradle.api.artifacts.Configuration#copy()[].
 ====
 
+=== Navigating the dependency graph
```

Comment:
Seems like we should consider showing the dependency tree of build scans here.

Reply:
I am not quite sure how a build scan would be relevant for explaining this API. We actually do want to walk the graph programmatically. The example I use here is that you are able to create a _different_ visualization than provided by a build scan e.g. an actual DAG. I think the story would be different if a build scan would provide such a visualization (especially if you could zoom in/out etc.) but at the moment it doesn't. What were your thoughts here?